includes/base/element-base.php

```diff
@@ -116,7 +116,7 @@ public function get_controls( $control_id = null ) {
 		return self::_get_items( $stack['controls'], $control_id );
 	}
 
-	public function add_control( $id, $args ) {
+	public function add_control( $id, array $args ) {
```

Comment:
This type of commit shouldn't be on a different PR though? It has nothing to do with the docs..